src/ecpair.js

```diff
@@ -178,4 +133,26 @@ ECPair.prototype.verify = function (hash, signature) {
   return ecdsa.verify(hash, signature, this.Q)
 }
 
+/**
+ * @deprecated
+ * Use {@see keyutil.privateKeyBufferToECPair} instead
+ * Will be removed in next major version
```

Comment:
How do you ensure that you don't forget this? ;)
Maybe add a FixMe with a ticket that you can track and search in the codebase for?

Reply:
trust me, the code haunts me ;) I'll add a ticket